code/datums/emergency_calls/contractor.dm

```diff
@@ -85,7 +85,7 @@
 	max_synths = 2
 
 /datum/emergency_call/contractors/covert
-	name = "Military Contractors (Covert) (Hostile to WY)"
+	name = "Military Contractors (Covert) (Friendly)"
```

Comment:
Why make this change to VAISO (the covert variant)? Corporate sabotage against WY is part of the reason they exist and their objectives still reflect this; "Assist USCMC forces in whatever way is possible, sabotage Weyland-Yutani efforts."

Reply:
Because covert when merged did not spawn as ERT. As they currently do they cause problems. Also sabotaging efforts does not require murder in the midst of an evac that would then cause a 3 way fight